test/wasm2c/spec/unwind.txt

```diff
@@ -1,5 +0,0 @@
-;;; TOOL: run-spec-wasm2c
```

Comment:
I think this file should stay actually, the `third_party/testsuite/unwind.wast` is not related to `try-unwind` AFAICT.

Reply:
Oh, thanks for the catch. I blindly searched for `unwind`... Restored.